model/src/main/resources/reference.conf

```diff
@@ -248,7 +248,7 @@ ort {
     mail {
       hostName = "localhost"
       username = user
-      password = secret
+      password = password
```

Comment:
If you change values here you also need to update `OrtConfigurationTest` accordingly.